contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -75,8 +84,16 @@ contract TACoChildApplication is ITACoRootToChild, ITACoChildApplication, Initia
         adjudicator = _adjudicator;
     }
 
+    function setActiveRituals(uint32[] memory _activeRituals) external {
```

Comment:
Should there be a restriction on who can call this?

Reply:
thank you, missed it!

Reply:
Contract is not ownable so I added `initializer` to restrict it

Reply:
^ Do we need updates to the deployment scripts for the upgrade today?

Reply:
yes and no, we can execute two steps upgrade on testnet, for the mainnet - yeah we want slight change in upgrade script